Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser.py

```diff
@@ -460,6 +504,35 @@ def revoke_user_session_command(client: MsGraphClient, args: Dict):
     return human_readable, None, None
 
 
+@suppress_errors_with_404_code
+def get_user_signins_command(client: MsGraphClient, args: Dict):
+    user = args.get('user', None)
+    ago = args.get('ago', '6 hours ago')
```

Comment:
```suggestion
    ago = args.get('time_ago', '6 hours ago')
```
Also why the default is 6 days?
And don't forget to update the argument name in the yml and readme as well